README.md

```diff
@@ -181,3 +181,6 @@ Meet the talented individuals who have contributed to Clean React App project:
   <img src="https://contrib.rocks/image?repo=rubemfsv/clean-react-app" />
 </a>
 
+## 📜 Code of Conduct
```

Comment:
Put it after the issues section, don’t forget to use the hr br tags according to the readme pattern

Reply:
Tks @rubemfsv!! I change this!